configuration/configuration_blackbox_test.go

```diff
@@ -89,7 +89,7 @@ func TestGetKeycloakEndpointSetByUrlEnvVaribaleOK(t *testing.T) {
 func TestGetKeycloakEndpointAdminDevModeOK(t *testing.T) {
 	resource.Require(t, resource.UnitTest)
 	t.Parallel()
-	checkGetKeycloakEndpointOK(t, "http://sso.demo.almighty.io/auth/admin/realms/fabric8", config.GetKeycloakEndpointAdmin)
+	checkGetKeycloakEndpointOK(t, "http://sso.prod-preview.openshift.io/auth/admin/realms/fabric8", config.GetKeycloakEndpointAdmin)
```

Comment:
Thanks @alexeykazakov , How are you handling the problem you had mentioned before - about --  no existing request context to derive the sso url from.. ?